### ISSUE TITLE
Extra blank lines when creating todotxt from empty file

### ISSUE THREAD

**Issue by LevonLu** [vonshednob/pytodotxt#9]
It happend when I try to creat a "Todotxt" from a empty file.

This is my test code which based on ["examples/adding_task.py"](https://github.com/vonshednob/pytodotxt/blob/master/examples/adding_task.py)

```python
"""Example how to add a task to an existing todo.txt file"""
import pytodotxt


todotxt = pytodotxt.TodoTxt("todo.txt")
todotxt.parse()

new_task = pytodotxt.Task("(A) Do some more stuff")
todotxt.add(new_task)

todotxt.save()

line_num = 1
for line in todotxt.lines:
    print(f"{line_num}: {line}")
    line_num += 1
```

Afterthis code run, "todo.txt" is this (one blank line and one task line and one blank line again)
```text

(A) Do some more stuff

```
And I used Windows 10 and Python 3.10

**Comment by vonshednob** [comment]
Nice find!

I can reproduce the issue and am fixing it now.

Thanks for reporting it!

**Comment by vonshednob** [comment]
Alright, fixed in 1.4.2.
